routes/home.go

```diff
@@ -0,0 +1,15 @@
+package routes
```

Comment:
Please move this and `router.go` into a single file called `routes.go`.

It's better to start with few files and let them grow bigger naturally, _then_ split them apart. This means that you are more likely to have sensible files as you are making them when needed rather than preemptively.

In some languages it's common to have one-class-per-file type of arrangements. In Go it's more idiomatic to group related types/structs/functions into a single file to avoid juggling hundreds of files all filled with like 10 lines of code.